packages/expo-dev-menu/android/src/main/java/expo/modules/devmenu/DevMenuActivity.kt

```diff
@@ -58,10 +62,11 @@ class DevMenuActivity : ReactActivity() {
       override fun getReactNativeHost() = DevMenuManager.getMenuHost()
 
       override fun getLaunchOptions() = Bundle().apply {
-        putBoolean("showOnboardingView", DevMenuManager.getSettings()?.isOnboardingFinished != true)
         putString("uuid", UUID.randomUUID().toString())
         putBundle("appInfo", DevMenuManager.getAppInfo())
         putBundle("devSettings", DevMenuManager.getDevSettings())
+        putBundle("menuPreferences", DevMenuManager.getMenuPreferences())
+        putBoolean("isSimulator", isSimulator)
```

Comment:
i'd suggest inverting the logic and calling this `isDevice` instead, to err on the side of precision since we tend to say "iOS simulators" and "android emulators"